selectrum.el

```diff
@@ -1366,15 +1366,14 @@ shadows correctly."
                  (let ((candidate-paths
                         (mapcar (lambda (path)
                                   (propertize
-                                   (file-name-base path)
+                                   (file-name-base
+                                    (file-name-sans-extension path))
                                    'selectrum-candidate-display-prefix
                                    (file-name-directory
                                     (file-name-sans-extension
                                      (selectrum--trailing-components
                                       num-components path)))
-                                   'fixedcase 'literal
-                                   'selectrum-candidate-full
-                                   path))
```

Comment:
Doesn't this make it so you can no longer choose which of several shadowed libraries by the same name you want to find? The current implementation is definitely wrong because it abuses the `selectrum-candidate-full` text property (with predictable results in terms of `TAB` being broken). But I think we could stick a different (internal to this function) text property onto each candidate, and read out the full path from that after `selectrum-read` returns.

Reply:
Oops sorry. 

> But I think we could stick a different (internal to this function) text property onto each candidate, and read out the full path from that after selectrum-read returns.

Nice idea will do that!

Reply:
Done